salt/modules/win_wusa.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Update files management via wusa.exe
+
+:maintainer:    Thomas Lemarchand
+:platform:      Windows
+:depends:       PowerShell
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
```

Comment:
`unicode_literals` needs to be included here from `__future__` as well for Py3 compatibility.